BootloaderCommonPkg/Library/ConsoleOutLib/ConsoleOutLibNull.c

```diff
@@ -0,0 +1,60 @@
+/** @file
```

Comment:
I think fixed ConsoleOuptutDeviceMask PCD will remove all the code from the ConsoleOutLib if we set it to 0.  Maybe we won't have to have NULL instance lib.

Reply:
Will remove the null library instance.